content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress gateway with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
+
+## Prerequisites
+
+cert-manager: Install cert-manager using [chart](https://github.com/helm/charts/tree/master/stable/cert-manager)
+
+## Check cert-manager installation
+
+1. Check if cert-manager was installed using Helm with the following command:
+
+{{< text bash >}}
+$ helm ls
+{{< /text>}}
+
+The output should be similar to the example below and show cert-manager with a `STATUS` of `DEPLOYED`:
+
+{{< text bash >}}
+NAME         	REVISION	UPDATED                 	STATUS  	CHART                    	APP VERSION 	NAMESPACE
+istio        	1       	Thu Oct 11 13:34:24 2018	DEPLOYED	istio-1.0.2              	1.0.2       	istio-system
+cert        	1       	Wed Oct 24 14:08:36 2018	DEPLOYED	cert-manager-v0.6.0-dev.2	v0.6.0-dev.2	istio-system
+{{< /text >}}
+
+## Create cluster issuer
+
+{{< warning_icon >}} Adapt cluster issuer provider with your own configuration, in our example we use `route53`
+
+{{< text yaml >}}
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-demo
+  namespace: kube-system
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: <REDACTED>
+    # Name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: letsencrypt-demo
+    dns01:
+      # Here we define a list of DNS-01 providers that can solve DNS challenges
+      providers:
+      - name: aws-dns
+        route53:
+          accessKeyID: <REDACTED>
+          region: eu-central-1
+          secretAccessKeySecretRef:
+            name: prod-route53-credentials-secret
+            key: secret-access-key
+{{< /text >}}
+
+## Create secret
+
+If you use provider `route53` you must provide secret in order to perform DNS ACME Validation.
+
+{{< text yaml >}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: prod-route53-credentials-secret
+type: Opaque
+data:
+  secret-access-key: <REDACTED BASE64>
+{{< /text >}}
+
+## Create Certificate
+
+Create your own certificate:
+
+{{< text yaml >}}
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: demo-certificate
+  namespace: istio-system
+spec:
+  acme:
+    config:
+    - dns01:
+        provider: aws-dns
+      domains:
+      - '*.mydemo.com'
+  commonName: '*.mydemo.com'
+  dnsNames:
+  - '*.mydemo.com'
+  issuerRef:
+    kind: ClusterIssuer
+    name: letsencrypt-demo
+  secretName: istio-customingressgateway-certs
+{{< /text >}}
+
+Have a look of `secretName`, it will be used in next section.
```

Comment:
Indent the text. Do not use future tense nor passive voice. To fix, I suggest:
```suggestion
Make a note of the value of `secretName` since a future step requires it.
```